docs/reference/high-availability/cluster-design.asciidoc

```diff
@@ -10,15 +10,12 @@ There is a limit to how small a resilient cluster can be. All {es} clusters
 require:
 
 * One <<modules-discovery-quorums,elected master node>> node
+* At least one node for each <<modules-node,role>>.
 * At least one copy of every <<scalability,shard>>.
 
-We also recommend adding a new node to the cluster for each
```

Comment:
More a requirement than a recommendation; added to the bullet-pointed list so it parallels the next one.